app/src/main/java/at/roteskreuz/stopcorona/screens/infection_info/InfectionInfoViewModel.kt

```diff
@@ -21,12 +21,9 @@ class InfectionInfoViewModel(
 ) : ScopedViewModel(appDispatchers) {
 
     fun observeInfectedContacts(): Observable<InfectedContactsViewState> {
-        return Observables.combineLatest(
-            infectionMessengerRepository.observeReceivedInfectionMessages(),
-            quarantineRepository.observeQuarantineState()
-        ).map { (messages, quarantineStatus) ->
+        return quarantineRepository.observeQuarantineState().map { quarantineStatus ->
             InfectedContactsViewState(
-                messages = messages,
+                messages = emptyList(),
```

Comment:
@mihbt Can you please check it also?

Reply:
@falkorichter , @dusanjencik  To know if we need to pass here additional data, we need to know what info we should display in this screen (https://zpl.io/aBxjWnm). Do you know how this screen looks like in the update for G/A?